docs(community): Add Captain's Log (#1699)
spinnaker/spinnaker.github.io: community/captains-log/index.md

---
layout: single
title:  "Captain's Log"
sidebar:
  nav: community
github_events:
  image_path: assets/images/graphs/GithubEvents_graph.png
  alt: "Graph of Github Events Since Open-Sourced"
activity:
  image_path: assets/images/graphs/ActivityByRepo_graph.png
  alt: "Graph of Github Activity by Repository Since Open-Sourced"
pr_authors:
  image_path: assets/images/graphs/PRAuthorsUnique_graph.png
  alt: "Graph of Unique PR Authors Since Open-Sourced"
---

You may have heard [success stories](/success-stories/) of enterprises moving from brittle deployments to continuous delivery with Spinnaker:
* [Airbnb uses Spinnaker](https://techbeacon.com/app-dev-testing/how-airbnb-scaled-its-migration-continuous-delivery-spinnaker) to migrate from monolith to service-oriented architecture.
* [SAP leverages Spinnaker](https://blog.spinnaker.io/pipeline-redemption-how-spinnaker-is-shaping-delivery-excellence-at-sap-3b3c931b4f63?) on its mission to run the world better.

> * [Pinterest uses Spinnaker](https://devops.com/devops-chat-ci-cd-velocity-for-large-monolithic-services-with-pinterest/) to boost productivity as it pioneers visual discovery.

Reviewer: "Pinterest boosts productivity with Spinnaker..."

Author: love it, updating

* [Mercari champions Spinnaker](https://speakerdeck.com/tcnksm/continuous-delivery-for-microservices-with-spinnaker-at-mercari) as a safeguard against deployment fear while releasing new services.
* [Salesforce adopted Spinnaker](https://engineering.salesforce.com/salesforce-speakers-at-spinnaker-summit-and-kubecon-2019-d968292fd681) to bake images for both Kubernetes and VMs, to support its complex delivery requirements.

### But what are the numbers? How big is the project? What’s the growth trajectory?

## Spinnaker's Trajectory

{%
  include
  figure
  image_path="./stats-2020-02-05-with-logo.png"
%}

> The Spinnaker community began as a partnership between Netflix and Google, eager to share the benefits of Continuous Delivery. It has grown into a vibrant Open Source project, attracting hundreds of companies to participate as it evolves, integrating unique use cases and tool chains. Check it out!

Reviewer: I like the call to action idea ("check it out"), but maybe needs a different phrase for a VP/C-suite level audience? 

Also not sure about the placement... Right in the middle of the "pitch" seems wonky.

Author: Yeah I went back and forth on this. The placement didn't look great to me with too much text at the top, I didn't want 2 paragraphs before getting to the meat. I can play with it more tho

Author: K @ttomsu I shortened the copy so it's just 2 lines and keeps the graphic "above the fold" to catch the eye. Thoughts on "Behold" instead of "check it out"? Is lol

### GitHub Events in Spinnaker Repositories

After steadily building momentum since its first open-source release in 2015, Spinnaker activity takes off:
{%
  include
  figure
  image_path="./github_events.png"
%}

### Contributions Per Company

> Building from a committed base of key orgs like Netflix, Google, Armory, OpsMx, and Amazon, 2019 saw significantly more contributions from end-user companies and new stakeholders:

Reviewer: s/orgs/organizations

Author: fixed thanks

---
layout: single
title:  "Captain's Log"
sidebar:
  nav: community
github_events:
  image_path: assets/images/graphs/GithubEvents_graph.png

>   alt: "Graph of Github Events Since Open-Sourced"

Reviewer 2: ```suggestion
  alt: "Graph of GitHub events since open-sourced"
```

Author: @dorbin do you think all of the titles should be lower-cased? Is this our style convention?

Reviewer 3: We decided that we're going to use the Google Style guide, and that uses sentence case for titles.

activity:
  image_path: assets/images/graphs/ActivityByRepo_graph.png

>   alt: "Graph of Github Activity by Repository Since Open-Sourced"

Reviewer 2: ```suggestion
  alt: "Graph of GitHub activity by repository since open-sourced"
```

pr_authors:
  image_path: assets/images/graphs/PRAuthorsUnique_graph.png

>   alt: "Graph of Unique PR Authors Since Open-Sourced"

Reviewer 2: ```suggestion
  alt: "Graph of unique PR authors since open-sourced"
```

---

You may have heard [success stories](/success-stories/) of enterprises moving from brittle deployments to continuous delivery with Spinnaker:
* [Airbnb uses Spinnaker](https://techbeacon.com/app-dev-testing/how-airbnb-scaled-its-migration-continuous-delivery-spinnaker) to migrate from monolith to service-oriented architecture.
* [SAP leverages Spinnaker](https://blog.spinnaker.io/pipeline-redemption-how-spinnaker-is-shaping-delivery-excellence-at-sap-3b3c931b4f63?) on its mission to run the world better.
* [Pinterest boosts productivity with Spinnaker](https://devops.com/devops-chat-ci-cd-velocity-for-large-monolithic-services-with-pinterest/) as it pioneers visual discovery.
* [Mercari champions Spinnaker](https://speakerdeck.com/tcnksm/continuous-delivery-for-microservices-with-spinnaker-at-mercari) as a safeguard against deployment fear while releasing new services.

> * [Salesforce adopted Spinnaker](https://engineering.salesforce.com/salesforce-speakers-at-spinnaker-summit-and-kubecon-2019-d968292fd681) to bake images for both Kubernetes and VMs, to support its complex delivery requirements.

Reviewer 2: s/adopted/adopts/ 
(parallel with the other ones)

### But what are the numbers? How big is the project? What’s the growth trajectory?

## Spinnaker's Trajectory

> Our community began as a partnership between Netflix & Google, eager to share the benefits of Continuous Delivery. Now it's a vibrant OSS project, attracting hundreds of companies to participate as it evolves, integrating unique use cases and tools. Behold!

Reviewer 2: I'm not sure "Behold!" is any more executivey than "Check it out!. Also, I'm not sure a call to action is needed here; The call to action is s basically just "Read on:"

But, *voice and tone* matter, and if you like the idea of having a casual tone inviting the reader to dig it, I prefer "Check it out" over Behold!" (and over "dig it" ;)

Author: haha okay, noted. I'd love to get more feedback on that word. at this point, behold is shorter, making the text block 2 lines which is good for my fold-centricness. I'll keep thinking and asking! @brian-armory, do you think a CTA is needed and which one(s) do you prefer?

Author: also @dorbin I agree on the potential ambiguity of "Captain's Log" so I've updated it to "Captain's Log: The State of Spinnaker" which will be good for SEO.

{%
  include
  figure
  image_path="./stats-2020-02-05-with-logo.png"
%}

### GitHub Events in Spinnaker Repositories

After steadily building momentum since its first open-source release in 2015, Spinnaker activity takes off:
{%
  include
  figure
  image_path="./github_events.png"
%}

### Contributions Per Company

Building from a committed base of key organizations like Netflix, Google, Armory, OpsMx, and Amazon, 2019 saw significantly more contributions from end-user companies and new stakeholders:
{%
  include
  figure
  image_path="./company_contributions.png"
%}

### Activity By Repository

> The Spinnaker ecosystem currently includes 44 repositories, including the microservices that deliver its core functionality and interface with deployment targets such as AWS, GCP, and Kubernetes. Also included: cleanup and monitoring tools, governance, community resources, and much more:

Reviewer 2: Add "documentation"?

Author: yes!

{%
  include
  figure
  image_path="./activity_by_repo.png"
%}

### Companies and Developers Contributing Each Week

Spinnaker has seen steadily increasing engagement from companies, and spikes of developer activity around community initiatives:
{%
  include
  figure
  image_path="./company_dev_count.png"
%}

### Myriad Pull Request Contributors

Unique authors contribute pull requests to Spinnaker repositories each week, building the project collaboratively over time.

{%
  include
  figure
  image_path="./pr_authors.png"
%}

### Visit Spinnaker DevStats for More!

> We've collected the data presented here from [Spinnaker's DevStats Dashboard](https://spinnaker.devstats.cd.foundation/), an awesome project growth [visualization and monitoring tool](https://github.com/cncf/devstats) built by CNCF engineers in collaboration with Kubernetes and other CNCF project communities. DevStats defines a “contribution” as a review, comment, commit, PR, or issue. Big thanks to Lukasz Gryglicki and the CNCF!

Reviewer 2: ```suggestion
The data presented here comes from [Spinnaker's DevStats Dashboard](https://spinnaker.devstats.cd.foundation/), an awesome project growth [visualization and monitoring tool](https://github.com/cncf/devstats) built by CNCF engineers in collaboration with Kubernetes and other CNCF project communities. DevStats defines a “contribution” as a review, comment, commit, PR, or issue. Big thanks to Lukasz Gryglicki and the CNCF!
```
